VagrantProvision.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+source $HOME/hoot/VagrantProvisionVars.sh
```

Comment:
Shouldn't this be `$HOOT_HOME/VagrantProvisionVars.sh`?  Latest in develop branch has `HOOT_HOME` defined first in provision scripts.

Reply:
@jbronn I fixed that a moment ago.